### PR TITLE
Proper fix for integration tests

### PR DESCRIPTION
**Pull request by vboulineau**
### What does this PR do?

Properly fix integration tests due to CircleCI migrating to newer Docker version with underlying Kernel using cgroupv2.

### Motivation

Bugfix.

### Additional Notes

Requires https://github.com/DataDog/datadog-agent-buildimages/pull/278.

### Reviewer's Checklist

- [x] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [x] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [x] Changed code has automated tests for its functionality.
- [x] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [x] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [x] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [x] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [x] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [x] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by ahmed-mez — APPROVED** [review]

**Comment by ahmed-mez** [comment]
Approved it too fast 😅 It's failing because of `TypeError: run() got an unexpected keyword argument 'cgroupns'`

**Review by mx-psi — APPROVED** [review]

**Review by djmitche — APPROVED** [review]

File: test/integration/util/kubelet/testdata/insecure-kubelet-compose.yaml [hunk 1]
```diff
@@ -1,18 +1,20 @@
 version: '2.3'
 services:
   kubelet:
-    image: "datadog/docker-library:hyperkube_1_8_3"
-    command: "/hyperkube
-        kubelet
-        --cloud-provider=''
+    image: "datadog/docker-library:kubelet_1.25"
```
Comment (clamoriniere) [inline test/integration/util/kubelet/testdata/insecure-kubelet-compose.yaml:4]: you are mixing hyperkube 1.18 with kubelet 1.25. Is it ok?

Comment (clamoriniere) [reply test/integration/util/kubelet/testdata/insecure-kubelet-compose.yaml:4]: ok forget about my previous message. files are used in different checks

**Review by clamoriniere — APPROVED** [review]

File: test/integration/util/kubelet/testdata/insecure-kubelet-compose.yaml (hunk 1 above)
Comment (vboulineau) [reply test/integration/util/kubelet/testdata/insecure-kubelet-compose.yaml:4]: It's not used in the same tests but yeah I could migrate to standard apiserver as well.

Comment (vboulineau) [reply test/integration/util/kubelet/testdata/insecure-kubelet-compose.yaml:4]: You cannot bump to `1.25` has the `--insecure-bind-address` flag has been removed.

**Pull request merged**